repository/src/main/java/org/apache/atlas/repository/util/FilterUtil.java

```diff
@@ -174,4 +178,31 @@ public static void addParamsToHideInternalType(SearchFilter searchFilter) {
         searchFilter.setParam(SearchFilter.PARAM_NOT_NAME, Constants.TYPE_NAME_INTERNAL);
         searchFilter.setParam(SearchFilter.PARAM_NOT_SUPERTYPE, Constants.TYPE_NAME_INTERNAL);
     }
+
+    public static boolean validateFilePath(String fileToImport) {
+        String allowedDirectory = "/var/app/allowed/";
```

Comment:
why allowed directory?

Reply:
I think we can remove this validation, this restricts file to be uploaded must be in a specific location always i.e. /var/app/allowed/